low/low.c

```diff
@@ -221,6 +250,14 @@ void eal_init(int argc, char *argv[], uint32_t burstSize)
 	free(argv[argc-1]);
 	free(argv);
 	BURST_SIZE = burstSize;
+	mbufStructSize = sizeof(struct rte_mbuf);
+	headroomSize = RTE_PKTMBUF_HEADROOM;
+	defaultStart = mbufStructSize + headroomSize;
+	zero128 = _mm_xor_ps(zero128, zero128);
+#ifdef AVX
+	zero256 = _mm256_xor_ps(zero256, zero256);
```

Comment:
If compiler allows initialization at variables declaration, these XORs are not needed.